{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/conftest.py

```diff
@@ -55,3 +57,8 @@ class Meta:
     username = factory.Sequence("user_{}@example.com".format)
     password = factory.PostGenerationMethodCall('set_password', 'foobar')
     socialaccount_set = factory.RelatedFactory(SocialAccountFactory, 'user')
+
+
+@pytest.fixture
+def anon_client():  # pragma: nocover
```

Comment:
This nocover can go away, if we're actually using this fixture.

Reply:
It's not currently used in the template, but seemed useful. I could comment it out entirely?

Reply:
I think it's useful enough to include, maybe just commented out is OK then.